app/ui/browser/views/navbar/location.jsx

```diff
@@ -28,6 +37,18 @@ const LOCATION_BAR_STYLE = Style.registerStyle({
   borderRadius: '2px',
 });
 
+const LOCATION_BAR_AUTOCOMPLETE_STYLE = Style.registerStyle({
+  position: 'absolute',
```

Comment:
I have lots of unanswered questions about that mock (like what happens with overflow), and trying to match it is proving tricky.  Are you OK with landing with the popup mode and then do styling updates in follow ups?